content/en/docs/azure/authentication-oidc.md

```diff
@@ -0,0 +1,212 @@
++++
+title = "Authentication using OIDC in Azure"
+description = "Authentication and authorization support through OIDC for Kubeflow in Azure"
+weight = 6
++++
+
+This section shows the how to setup Kubeflow with authentication and authorization support through OIDC in Azure using [Azure Active Directory](https://azure.microsoft.com/en-us/services/active-directory/).
```

Comment:
Let's still use "to set up" ([source](https://www.merriam-webster.com/dictionary/setup)) here, you may have missed it:

```suggestion
This section shows the how to set up Kubeflow with authentication and authorization support through OIDC in Azure using [Azure Active Directory](https://azure.microsoft.com/en-us/services/active-directory/).
```